assets/js/components/surveys/SurveyQuestionMultiSelect.js

```diff
@@ -0,0 +1,179 @@
+/**
+ * SurveyQuestionMultiSelect component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import keyBy from 'lodash/keyBy';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../Button';
+import Checkbox from '../Checkbox';
+import SurveyHeader from './SurveyHeader';
+import { TextField, Input } from '../../material-components';
+
+const MAXIMUM_CHARACTER_LIMIT = 100;
+
+// Why have this rule enabled if we get this from the APIs *and* send back to the APIs like this?
+/* eslint-disable camelcase */
```

Comment:
From the AC it looks like this component will be receiving snake-case props.

If we don't allow this then we should convert this when we receive it from the API

Reply:
It's too late to do this now as we're doing this everywhere, so I'm just disabling this rule per file :smile:

Reply:
We tend to disable it per-line… which yes is more effort at first but prevents us from making the mistakes elsewhere. Can you change it to be per-line?

Reply:
Hmm, while I see the IB mentions using `max_choices`, presumably because that's what the API response looks like, let's have our actual props be the usual case style eg `maxChoices`. Especially as we're manually providing them I think that's fine.

In this case the IB wasn't perfect and I think it's fine to diverge from it 🙂

Reply:
I've just put this in 12 times :grimacing:   I don't see this lint rule helping at all but it's done now :see_no_evil: